### PR TITLE
Updated README.md to link to RELEASING.md and LOCALKUBE_RELEASING.md

### PR DESCRIPTION
**Pull request by aaron-prindle**

### CODE REVIEW

**Comment by dlorenc** [comment]
@minikube-bot retest this please

File: README.md [hunk 1]
```diff
@@ -379,6 +379,12 @@ git add --all
 git commit -m "Manual changes to update Kubernetes to foo"
 ```
 
+## # Steps to Release Minikube
+The steps to release minikube can be found at [RELEASING.md](https://github.com/aaron-prindle/minikube/blob/master/RELEASING.md)
```
Comment (jimmidyson) [inline README.md]: Links to wrong repo - you can just use relative links to RELEASING.md

File: README.md [hunk 2]
```diff
@@ -379,6 +379,12 @@ git add --all
...
+
+## # Steps to Release Localkube
+The steps to release localkube can be found at [LOCALKUBE_RELEASING.md](https://github.com/aaron-prindle/minikube/blob/master/LOCALKUBE_RELEASING.md)
```
Comment (jimmidyson) [inline README.md]: Links to wrong repo - you can just use relative links to LOCALKUBE_RELEASING.md

File: README.md [hunk 3]
```diff
@@ -379,6 +379,12 @@ git add --all
 git commit -m "Manual changes to update Kubernetes to foo"
 ```
 
+## # Steps to Release Minikube
```
Comment (dlorenc) [inline README.md]: Formatting looks  a bit off, you don't need the third '#'.

**Comment by jimmidyson** [comment]
Good documentation - can we turn this in to a Makefile target, i.e. `make localkube-release`?

**Comment by dlorenc** [comment]
Yeah we should automate the releases for both minikube and localkube. Opened https://github.com/kubernetes/minikube/issues/361 for that.

**Comment by dlorenc** [comment]
LGTM

**Pull request merged**